search_test unit test breaking master intermittently

On occasion, this is happening.

```
HELM_HOME=/no/such/dir go test  -run . ./cmd/... ./pkg/...
ok  	k8s.io/helm/cmd/helm	0.810s
ok  	k8s.io/helm/cmd/helm/downloader	0.093s
ok  	k8s.io/helm/cmd/helm/helmpath	0.014s
ok  	k8s.io/helm/cmd/helm/installer	0.103s
ok  	k8s.io/helm/cmd/helm/resolver	0.033s
--- FAIL: TestAddRepo_Sort (0.00s)
	search_test.go:146: Expected "1.2.3", got "1.2.2"
FAIL
FAIL	k8s.io/helm/cmd/helm/search	0.029s
ok  	k8s.io/helm/cmd/helm/strvals	0.015s
ok  	k8s.io/helm/cmd/tiller	0.147s
ok  	k8s.io/helm/pkg/chartutil	0.114s
ok  	k8s.io/helm/pkg/engine	0.032s
ok  	k8s.io/helm/pkg/helm	0.042s
ok  	k8s.io/helm/pkg/ignore	0.020s
ok  	k8s.io/helm/pkg/kube	0.146s
ok  	k8s.io/helm/pkg/lint	0.035s
ok  	k8s.io/helm/pkg/lint/rules	0.034s
ok  	k8s.io/helm/pkg/lint/support	0.022s
ok  	k8s.io/helm/pkg/plugin	0.016s
?   	k8s.io/helm/pkg/proto/hapi/chart	[no test files]
?   	k8s.io/helm/pkg/proto/hapi/release	[no test files]
?   	k8s.io/helm/pkg/proto/hapi/services	[no test files]
?   	k8s.io/helm/pkg/proto/hapi/version	[no test files]
ok  	k8s.io/helm/pkg/provenance	0.542s
ok  	k8s.io/helm/pkg/releaseutil	0.023s
ok  	k8s.io/helm/pkg/repo	0.085s
ok  	k8s.io/helm/pkg/repo/repotest	0.050s
ok  	k8s.io/helm/pkg/storage	0.144s
ok  	k8s.io/helm/pkg/storage/driver	0.150s
ok  	k8s.io/helm/pkg/tiller	0.122s
ok  	k8s.io/helm/pkg/tiller/environment	0.115s
ok  	k8s.io/helm/pkg/timeconv	0.014s
?   	k8s.io/helm/pkg/tlsutil	[no test files]
ok  	k8s.io/helm/pkg/urlutil	0.020s
ok  	k8s.io/helm/pkg/version	0.022s
make: *** [test-unit] Error 1
```

Initially reported by @larryrensing

--- a/cmd/helm/search/search_test.go
+++ b/cmd/helm/search/search_test.go
@@ -139,6 +139,7 @@ func TestAddRepo_Sort(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+	SortScore(sr)
 
 	ch := sr[0]
 	expect := "1.2.3"

